CTP/shell/init_path/init.sh

```diff
@@ -75,6 +75,16 @@ function get_broker_port_from_shell_config
   echo $port
 }
 
+function get_cubrid_port_id(){
+    local port_id=`grep -i "cubrid_port_id" $CUBRID/conf/cubrid.conf | awk -F '=' '{print $2}'`
```

Comment:
```suggestion
    local port_id=`ini.sh -s "common" $CUBRID/conf/cubrid.conf cubrid_port_id`
```
주석처리 등이 고려돼야 할 것 같습니다. 이런 경우 사용되는 ini.sh를 이용하는 것이 좋을 것 같습니다.

Reply:
확인해본 결과, cubrid.conf에서 해당 파라미터가 주석처리된 경우.
server start시 default값이 지정되는 것을 확인할 수 있었습니다.

현재 파라미터를 찾지 못한 경우, 아래의 if문에서 1523(default)을 return하도록 처리해두어, 로직상에 문제는 없을 것 같습니다.
하지만 기존에 활용되고 있는 ini.sh을 사용하는 것이 code 유지보수에 더욱 도움이 될 것 같습니다.

리뷰 감사합니다.

Reply:
해당 내용 반영했습니다